e2e/testcafe-devextreme/tests/accessibility/gallery.ts

```diff
@@ -0,0 +1,46 @@
+import { Properties } from 'devextreme/ui/gallery.d';
+import url from '../../helpers/getPageUrl';
+import { testAccessibility, Configuration } from '../../helpers/accessibility/test';
+import { Options } from '../../helpers/generateOptionMatrix';
+
+fixture.disablePageReloads`Accessibility`
+  .page(url(__dirname, '../container.html'));
+
+const gallery = [
+  {
+    imageAlt: 'Image 1',
+    imageSrc: '../../images/1.jpg',
+  },
+  {
+    imageAlt: 'Image 2',
+    imageSrc: '../../images/2.jpg',
+  },
+  {
+    imageAlt: 'Image 3',
+    imageSrc: '../../images/3.jpg',
+  },
+];
+
+const options: Options<Properties> = {
+  height: [300],
+  width: [300],
+  dataSource: [gallery],
+  loop: [true, false],
+  swipeEnabled: [true, false],
+  stretchImages: [true, false],
```

Comment:
We can skip this option because we don't care if the images are stretched or not.